conf/modules.config

```diff
@@ -544,6 +544,47 @@ process {
             mode: params.publish_dir_mode,
         ]
     }
+
+}
+
+//
+// Score and rank structural variants
+//
+
+process {
+    withName: '.*RANK_VARIANTS_SV:GENMOD_ANNOTATE' {
+        ext.args = " --annotate_regions "
```

Comment:
We should add the genome build. 
maybe one could do something like:
```
ext.args = [
  '--annotate_regions',
  params.genome == 'GRCh37' ? '--genome-build 37' : '--genome-build 38',
].join(" ").trim()
```

Reply:
Sure! I was thinking about having it in hasta.config, but having it here is probably easier.